server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/analysis/Project.java

```diff
@@ -77,15 +86,16 @@ public String toString() {
     sb.append("uuid='").append(uuid).append('\'');
     sb.append(", key='").append(key).append('\'');
     sb.append(", name='").append(name).append('\'');
+    sb.append(", description=").append(toString(this.description));
     sb.append('}');
     return sb.toString();
   }
 
-  public static Project copyOf(Component component) {
-    return new Project(component.getUuid(), component.getKey(), component.getName());
+  private static String toString(@Nullable String s) {
+    if (s == null) {
+      return null;
+    }
+    return '\'' + s + '\'';
   }
 
-  public static Project copyOf(ComponentDto component) {
```

Comment:
What was wrong with these `copyOf` methods?

Reply:
they were used only in unit tests and I'm pretty sure introduced in production code for this only purpose => delete